core/connection/manager.go

```diff
@@ -157,7 +169,25 @@ func (manager *connectionManager) startConnection(consumerID identity.Identity,
 		return err
 	}
 
-	sessionID, sessionConfig, err := session.RequestSessionCreate(dialog, proposal.ID, sessionCreateConfig)
+	messageChan := make(chan balance.Message, 1)
+
+	// TODO: load initial promise state
```

Comment:
We could have a separate ticket for that, we could even link that in such TODO comment.

Reply:
https://github.com/mysteriumnetwork/node/issues/714